twilight-gateway/src/shard.rs

```diff
@@ -542,7 +542,12 @@ impl Shard {
                     TungsteniteMessage::Close(None)
                 }
                 NextMessageFutureOutput::SendHeartbeat => {
-                    self.heartbeat(None)
+                    // Start the heartbeat sequence at 0 if a `Session` is
+                    // unavailable (created after having reeived a `Ready`
+                    // event, which happens after identifying). Note that
+                    // there's no officially documented default value.
```

Comment:
The official default is `null`.
Reference: https://discord.com/developers/docs/topics/gateway#heartbeat-interval